.stylelintrc.json

```diff
@@ -14,7 +14,7 @@
     "order/properties-alphabetical-order": true,
     "no-descending-specificity": null,
     "selector-class-pattern": [
-      "^aa-(?:[A-Z][a-z]+)+(?:--[a-z]+(?:[A-Z][a-z]+)?)?$"
+      "^aa(-(?:[A-Z][a-z]+Plugin))?-(?:[A-Z][a-z]+)+(?:--[a-z]+(?:[A-Z][a-z]+)?)?$"
```

Comment:
Weird that we don't allow proper pascal case for the names

Reply:
We do for the element, the naming convention is BEM-like:

- `aa-` prefix
- Pascal case element (e.g., `ItemLabel`)
- Camel case modifier (e.g., `active`, `grayedOut`) prefixed with `--`

What you can't have it multiple elements (e.g., `aa-Item-Label`) which goes against BEM. But since we have plugins, we need to prefix their styles, which is what this change does.

Reply:
could also consider that the plugin classes aren't meant to clash with official ones and leave it as `aa-` instead of `aa-Panel-`

Reply:
It's not so much for clashing but easy identification; but still, if we have new styles in the classic theme later on we could easily forget about plugin styles and write clashing styles.

Any opinion @Shipow?

Reply:
I'm fine with the regex, it just need to be obvious that we are under a plugin.

Reply:
The class names are `aa-TagsPlugin-xxx` so I think it is indeed.